src/DataSet.h

```diff
@@ -149,10 +160,10 @@ class DataSet {
   
   void readPsm(const std::string& line, const unsigned int lineNr,
                const std::vector<OptionalField>& optionalFields, 
-               FeatureMemoryPool& featurePool);
+               FeatureMemoryPool& featurePool, std::string decoyPrefix="_rev");
```

Comment:
Don't you mean "rev_", also not sure if this is a good default value.

Reply:
FIxed name. However, I'm not sure either. We should perhaps discuss this on a later meeting.

Reply:
I have made this a required argument in 5fc8248